frontend/src/pages/CourseEditor/utils.ts

```diff
@@ -18,11 +15,8 @@ export const getPointsArr = (maxPoints: number): number[] => {
 };
 
 export const formatFieldValue = (value: string): string => {
-  let formattedValue = value.replace(TRAILING_SPACES_REGEX, ' ').trim();
-  if (formattedValue.length > EMPTY_LENGTH) {
-    formattedValue =
-      formattedValue[FIRST_LETTER_INDEX].toUpperCase() + formattedValue.slice(SECOND_LETTER_INDEX);
-  }
+  const formattedValue = value.replace(TRAILING_SPACES_REGEX, ' ').trim();
+
```

Comment:
! it's not the same
this util not only remove spaces, but also replace first letter with uppercase
on your branch this functinality is broken, so replace it with old implementation
<img width="776" alt="image" src="https://user-images.githubusercontent.com/44285339/176198125-2de0af9f-74d0-4177-a11d-cccf5a458cd1.png">

Reply:
Can we do it only on submit? also link shouldn't transform to uppercase as it's invalid format for them

Reply:
@SaXaPhonist  you can create custom onBlur for link fields, it should be formatted onBlur, not onSubmit

Reply:
i've made on submit already

Reply:
why it should be formated onBlur? onBlur more often then onSubmit

Reply:
answered below